src/build-data/makefile/header.in

```diff
@@ -19,7 +19,7 @@ CLI_FLAGS      = $(CXXFLAGS) $(WARN_FLAGS)
 TEST_FLAGS     = $(CXXFLAGS) $(WARN_FLAGS)
 
 SCRIPTS_DIR    = %{scripts_dir}
-INSTALLED_LIB_DIR = %{destdir}/%{libdir}
+INSTALLED_LIB_DIR = %{libdir}
```

Comment:
Just verified that this is the dir that applications linking against Botan will use to find the library at runtime, so we need `%{prefix}/%{libdir}`. This is an OS X variable.